Source/WebGPU/WGSL/EntryPointRewriter.cpp

```diff
@@ -274,7 +295,7 @@ void EntryPointRewriter::visit(Vector<String>& path, MemberOrParameter&& data)
 void EntryPointRewriter::appendBuiltins()
 {
     for (auto& data : m_builtins) {
-        m_functionDecl.parameters().append(makeUniqueRef<AST::ParameterValue>(
+        m_function.parameters().append(makeUniqueRef<AST::ParameterValue>(
```

Comment:
Renames are a great opportunity for a standalone patch 😅

Reply:
Fair enough, I'll move it into a separate patch